app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -150,7 +150,10 @@ private void setupDrawerContent(NavigationView navigationView) {
 
         menu.setGroupCheckable(GROUP_ID, true, true);
 
-        menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME)).setChecked(true);
+        MenuItem selectedItem = menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME));
```

Comment:
the readability of this is still not really good.

Could you split this up into two lines for getting the string and then finding the item?